docs/development/addon-installer.md

```diff
@@ -90,11 +90,13 @@ Additional functionality can be installed using the following guidelines:
         public function install()
         {
             parent::install();
+            return TRUE;
```

Comment:
@RK311y I think these lines should read `return parent::install();` to respect the result of addon  installer service execution;

And as extra note, since we are trying to follow PSR-12, `true` needs to be lowercase